game/game/gamescript.cpp

```diff
@@ -682,43 +672,42 @@ World &GameScript::world() {
   return *owner.world();
   }
 
-Daedalus::GEngineClasses::C_Focus GameScript::getFocus(const char *name) {
-  Daedalus::GEngineClasses::C_Focus ret={};
-  auto id = vm.getDATFile().getSymbolIndexByName(name);
-  if(id==size_t(-1))
-    return ret;
-  vm.initializeInstance(ret, id, Daedalus::IC_Focus);
-  vm.clearReferences(ret);
-  return ret;
+std::shared_ptr<phoenix::c_focus> GameScript::getFocus(const char *name) {
```

Comment:
`const char*` can be replaced to `string_view`

Reply:
Also `C_Focus` consume probably has no check for null and generally has no good way to handle it